YDotNet/Document/Transactions/Transaction.cs

```diff
@@ -180,7 +182,7 @@ public TransactionUpdateResult ApplyV1(byte[] stateDiff)
     /// <returns>The result of the update operation.</returns>
     public TransactionUpdateResult ApplyV2(byte[] stateDiff)
     {
-        return (TransactionUpdateResult)TransactionChannel.ApplyV2(Handle, stateDiff, (uint)stateDiff.Length);
+        return (TransactionUpdateResult) TransactionChannel.ApplyV2(Handle, stateDiff, (uint) stateDiff.Length);
```

Comment:
We should have a editorconfig or something for the space here. Because we get inconsistent suggestions by our IDEs